src/assets/layout.css

```diff
@@ -15,22 +15,29 @@
 }
 
 .mave-screen-title-bar {
-  align-items: center;
+  align-items: flex-end;
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
   font-size: 28px;
   padding: 0 0 5px 0;
   border-bottom: 1px solid #ccc;
   margin: 10px 0;
+  overflow: hidden; /* Prevents layout shifting */
 }
 
 .mave-screen-title {
-  flex: 0 0 auto;
+  flex: 1 1 auto; /* Allow it to grow when possible */
+  display: flex;
+  flex-direction: column;
+  word-break: break-word;
+  white-space: normal;
+  overflow: visible;
 }
 
 .mave-screen-title-controls {
-  flex: 0 0 auto;
+  flex: 0 1 auto; /* Only take space if present */
+  align-self: flex-end;
+  display: flex;
 }
 
 .mave-screen-title-controls .p-button {
```

Comment:
In this element, I like adding a line `width: max-content`. That makes the buttons maintain the full width of their text


<img width="1051" alt="Screenshot 2025-03-20 at 11 05 49 AM" src="https://github.com/user-attachments/assets/d03f1cf8-3fc7-44cf-8c02-21a07bb5d8de" />